src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs

```diff
@@ -48,6 +48,17 @@ private IGeoMasterClient InitClient()
             return geoMasterClient;
         }
 
+        public string RemovePIIFromSettings(string content)
+        {
+            // Mask SAS Uri
+            if (Regex.Match(content, @"https*:\/\/.*\.core\.windows\.net.*sig=.*", RegexOptions.IgnoreCase).Success)
```

Comment:
Nitpick.. 
Although the above regex does the job as it matches all the requirements of a webapp, possibly use this following Regex ?

https*:\/\/[\w\.]*[\w]+\.core\.windows\.net\?.*sig=.*